tests/ui/lint/command-line-lint-level-specific-lint.rs

```diff
@@ -0,0 +1,11 @@
+// compile-flags: -A warnings -W unused-variables
```

Comment:
looks like we have no ui tests for just `-A warnings`. Please add some.

Reply:
Added a UI test for `-A warnings`.